src/data/faq.json

```diff
@@ -2171,66 +2171,304 @@
                 }
             ],
             "E": [
+                {
+                    "term": "Effective validity",
+                    "anchor": "effective_validity",
+                    "description": "We refer to the duration of vaccination protection as 'effective validity'. The effective validity of the vaccination thus counts from the last vaccination date. While in Germany the professional validity is currently set at one year, the validity of the vaccination protection in other countries may differ from this. In the case of a <a href='#glossary_recovery_certificate'>recovery certificate</a>, the effective validity is set at 'date of the person's first positive <a href='#glossary_pcr_test'>PCR test</a> result' plus 180 days, according to the EU eHealth Network."
+                },
+                {
+                    "term": "Error report",
+                    "anchor": "error_report",
+                    "description": "A detailed report of a malfunction that has occurred, which is sent to the responsible developers so that they can fix the described error."
+                },
                 {
                     "term": "EU DCC",
                     "anchor": "eu_dcc",
                     "description": "See <a href='#glossary_dcc'>DCC</a>"
+                },
+                {
+                    "term": "European Federation Gateway Service (EFGS)",
+                    "anchor": "efgs",
+                    "description": "A central infrastructure that is used to exchange and check <a href='#glossary_diagnosis_key'>diagnosis keys</a> across national borders."
+                },
+                {
+                    "term": "Event registration",
+                    "anchor": "event_registration",
+                    "description": "Register for an event or a visit to a location."
+                },
+                {
+                    "term": "Exposure Detection",
+                    "anchor": "exposure_detection",
+                    "description": "List of short-lived random Bluetooth IDs received and temporarily stored in the operating system memory. The encounter record is read as part of the risk determination during the risk check. All short-lived random Bluetooth IDs older than 14 days are automatically deleted. The encounter record can also be deleted in its entirety at any time by individuals using the app. When the CWA is uninstalled, the encounter record is automatically deleted."
+                },
+                {
+                    "term": "Exposure Notification Express (ENE)",
+                    "anchor": "ene",
+                    "description": "Starting with iOS 13.7, individuals can be notified of potential exposure to COVID-19 without the need for a dedicated exposure notifications app. This app-free feature is called Exposure Notifications Express and is only available if a public health authority (PHA) supports it."

```

Comment:
Should we mention here that the German public health authorities do not support it?

Reply:
@Ein-Tim 
> Should we mention here that the German public health authorities do not support it?

Good comment!

@larswmh 
In fact, since there is already an extensive FAQ entry on ENE https://www.coronawarn.app/en/faq/#ene
I would remove it from the glossary, or just refer to the FAQ entry.

Reply:
Changed it so it links to the FAQ article now, thanks!